interface-definitions/interfaces-ethernet.xml.in

```diff
@@ -268,6 +268,37 @@
             </properties>
             <defaultValue>auto</defaultValue>
           </leafNode>
+          <node name="ring-buffers">
```

Comment:
Can we please use singular `ring-buffer` over plural `ring-buffers`? I guess this is what the CLI guideline says.

Reply:
Fixed.